src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
We are happy to accept the help. Currently we are not taking care even to allocate aligned / padded memory for SIMD